.github/bin/check-package-version.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Need to add the Cylc copyright header to the top of this file.

https://github.com/cylc/cylc-flow/blob/master/setup.py#L4-L18